### ISSUE TITLE
map flickers (tiles are reloaded/recomputed) when map center passes the date line

### ISSUE THREAD

**Issue by ThFischer** [tangrams/tangram#589]
**TO REPRODUCE THE ISSUE, FOLLOW THESE STEPS:**
Openthe simple demo: [http://tangrams.github.io/simple-demo/](http://tangrams.github.io/simple-demo/)
Execute this code in the JS console:
```javascript
zoom = 3;
lng = 170;
map.setView({lat: 0, lng: lng}, zoom);
setInterval(
  function() {
    lng = lng < 180 ? 190 : 170;
    map.setView({lat: 0, lng: lng}, zoom);
  },
  3000
);
```
The map should be quite large (e.g. 2000 x 1000 px) to see the flickering. It may also help to change the zoom value (3) to a different level until you can see the flickering

**RESULT:**

Map tiles are reloaded/recomputed every time when the map center passes the international date line (180° longitude). That causes a map flickering.

**EXPECTED RESULT:**
Tiles shouldn't be reloaded/recomputed, map shouldn't flicker.

**ENVIRONMENT:**
OS X 10.11.6
* Firefox 55.0.3
* Chrome 60.0.3112.113
* Safari 10.1.2

**Comment by meetar** [comment]
Thanks for the issue, @ThFischer!

**Comment by meetar** [comment]
This has been fixed since Tangram v0.12, simple-demo updated with https://github.com/tangrams/simple-demo/commit/ecc4ec62c16e5d6710f9e546a45ab0f2aa5bd810
